CHANGELOG.md

```diff
@@ -24,6 +24,8 @@
   at `go.uber.org/dig`.
 - **[Breaking]** Pass a tracer the `uhttp/uhttpclient` constructor explicitly, instead
   of using a global tracer. This will allow to use http client in parallel tests.
+- **[Breaking]** Introduce a config loader, this will allow to override config loading
```

Comment:
is it actually breaking? if so, we should add some notes as to what the user should expect to do to migrate